### PR TITLE
added todo

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following
1. Todo Option, user can add todo, clear todo once done
2. Todo will persist on page reload as well and the state of todo will also persist on page reload
3. Kindly check the gif for reference

Extensry-_4_

### CODE REVIEW

File: src/components/Todo/Todo.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.todo-container {
+    width: 20vw;
+    background-color: #0f172a;
```
Comment (SanketDhabarde) [inline src/components/Todo/Todo.css]: Suggestion: Stay consistent if you are using a variable for some color and a hex for some color.

**Review by SanketDhabarde — APPROVED** [review]
- Code looks good
- Like the use of CSS combinator

**Review by jeetbhalani15 — APPROVED** [review]
Code &  UI looks great

File: src/components/Todo/Todo.css (hunk 1 above)
Comment (prakashsakari) [reply src/components/Todo/Todo.css]: thank you for pointing this out, I somehow missed it

Comment (prakashsakari) [reply src/components/Todo/Todo.css]: thanks, updated

**Pull request merged**